packages/terra-application-layout/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "terra-application-layout",
+  "main": "lib/ApplicationLayout.js",
+  "version": "0.0.0",
+  "description": "The Terra Application Layout is a responsive, themeable layout for building applications.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-application-layout",
+    "ApplicationLayout",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-app-delegate": "^2.0.0",
+    "terra-base": "^3.0.0",
+    "terra-props-table": "^2.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-app-delegate": "^2.0.0",
+    "terra-base": "^3.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-application-links": "^2.1.0",
+    "terra-application-header-layout": "^2.1.0",
+    "terra-application-name": "^2.1.0",
+    "terra-application-menu-layout": "^2.1.0",
+    "terra-application-utility": "^0.x",
+    "terra-button": "^2.2.0",
+    "terra-icon": "^2.2.0",
+    "terra-navigation-layout": "^2.1.0",
+    "terra-popup": "^2.2.0",
+    "terra-clinical-action-header": "^2.1.0",
```

Comment:
Could we avoid a clinical dependency?

Reply:
Possibly come back later and update it once action header exists in core.

Reply:
When do we think that's happening?

Reply:
Q2